flags/middleware.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import absolute_import
+
+from flags.settings import get_flags
+
+
+class FlagConditionsMiddleware(object):
```

Comment:
Could this inherit from [`django.utils.deprecation.MiddlewareMixin`](https://docs.djangoproject.com/en/2.1/topics/http/middleware/#upgrading-pre-django-1-10-style-middleware) to better-conform to Django 1.10+ middleware? I'm not sure the tests properly cover the case when this is invoked as part of the normal Django request-response cycle, and from the docs it looks like `process_request` is no longer called directly in recent versions if `MiddlewareMixin` isn't used.

Reply:
It doesn't seem harmful though, and all the fashionable middleware is doing it. It's also as good a reason as any to drop Django 1.8 support too.